src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Shouldn't import statements be explicitly stated instead?

Consider listing out each one explicitly.

Reply:
I agree with all your comments, thanks for helping me spot errors wrt coding standard. Cheers!